ast/builtins.go

```diff
@@ -959,6 +961,44 @@ The set of regex symbols is limited for this builtin: only ` + "`.`, `*`, `+`, `
  */
 var stringsCat = category("strings")
 
+var AnyPrefixMatchesAnyString = &Builtin{
+	Name:        "any_prefix_matches_any_string",
```

Comment:
Shouldn't the order of arguments be reversed based on the function name?

Reply:
I've used the same order as startswith and endswith to stay consistent and I think that's the way to go. I'm happy to change the name though. Do you have a better idea for it maybe @marcinwyszynski?

Reply:
I've renamed the function, which now better matches the order of arguments.